src/VirtoCommerce.CatalogModule.Web/Scripts/blades/item-detail.js

```diff
@@ -65,10 +65,31 @@ angular.module('virtoCommerce.catalogModule')
             return !angular.equals(blade.item, blade.origItem) && blade.hasUpdatePermission();
         }
 
+        function isEmptyValue(value) {
+            return value == null || value === '';
+        }
+
+        function isValidQuantity(item) {
+

```

Comment:
item.minQuantity and item.maxQuantity both can be null or not defined, please also consider this requirement.

Reply:
IsEmptyValue is consider undefined and null cases, if you test it with null or undefined you can see that the method IsValidQuantity is returning true

Reply:
What about whitespaces? `' '`